renderdoc/data/glsl/debuguniforms.h

```diff
@@ -89,7 +89,7 @@ struct Vec4u
 #ifndef OPENGL_ES
 #define PRECISION
 #else
-#define PRECISION mediump
+#define PRECISION highp
```

Comment:
Ping @pantosj - anything to consider here with changing this? I don't understand the impact.